# Config file for f110_reactive_methods package

# Lidar Scan Truncation Coverage Angle
truncated_coverage_angle : 3.14

# High Distance rejection threshold
max_accepted_distance : 3.0

# Ranges Smoothing Filter Size
smoothing_filter_size : 3

# Safety Bubble Threshold
bubble_radius : 0.7

# Velocity
error_based_velocities : {low: 2.0, medium: 1.5, high: 1.0}


